Aardvark/Aardvark.h

```diff
@@ -2,39 +2,33 @@
 //  Aardvark.h
 //  Aardvark
 //
-//  Created by Dan Federman on 10/4/14.
-//  Copyright 2014 Square, Inc.
-//
-//  Licensed under the Apache License, Version 2.0 (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-//  Unless required by applicable law or agreed to in writing, software
-//  distributed under the License is distributed on an "AS IS" BASIS,
-//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//  See the License for the specific language governing permissions and
-//  limitations under the License.
+//  Created by Evan Kimia on 10/22/15.
+//  Copyright © 2015 Square, Inc. All rights reserved.
 //
 
-#import <Aardvark/ARKBugReporter.h>
-#import <Aardvark/ARKEmailBugReporter.h>
+#import <UIKit/UIKit.h>
 
+//! Project version number for Aardvark.
+FOUNDATION_EXPORT double AardvarkVersionNumber;
```

Comment:
Interesting. Why is this needed? We didn't need this when we converted Valet over. Can you link me to documentation?

Thanks for helping out a Carthage newb :)